opcodes/dxil/dxil_ray_tracing.cpp

```diff
@@ -24,6 +24,28 @@
 
 namespace dxil_spv
 {
+static spv::Id emit_temp_storage_copy(Converter::Impl &impl, const llvm::Value *value, spv::StorageClass storage)
+{
+	// Make a new temporary variable for the ray payload/callable data.
+	auto *pointer_type = llvm::cast<llvm::PointerType>(value->getType());
```

Comment:
There should be a unique payload variable per type + storage to avoid bloating up things.

Reply:
Resolved locally.